server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -229,7 +229,7 @@ static TransportVersion def(int id) {
      * Reference to the minimum transport version that can be used with CCS.
      * This should be the transport version used by the previous minor release.
      */
-    public static final TransportVersion MINIMUM_CCS_VERSION = V_8_11_X;
+    public static final TransportVersion MINIMUM_CCS_VERSION = ENRICH_ELASTICSEARCH_VERSION_REMOVED;
```

Comment:
As of #104937 , I think we'd need to use `V_8_12_0` here, since `ENRICH_ELASTICSEARCH_VERSION_REMOVED` got collapsed away.

Reply:
This is superceded by #105141 anyway...

Reply:
8.13 has just been branched, so we need to update this now